recipes/default.rb

```diff
@@ -17,5 +17,22 @@
 # limitations under the License.
 #
 
+# if trying to use opcache with an incompatible version of php (default pkg)
+# and using an unsupported version of php. use_ius must be true to install a valid php interpreter.
+if node['osl-php']['use_opcache'] && node['php']['version'].to_f < 5.5
+  raise 'Must use PHP >= 5.5 with ius enabled to use Zend Opcache.  Try adding '\
+        "'node.default['osl-php']['use_ius'] = true' and install a proper version of php."
+end
+
+node.normal['osl-php']['php_packages'] += %w(opcache) if node['osl-php']['use_opcache']
```

Comment:
The more I look at this, the more I think it might be better to move this all into an ``opcache`` recipe like we have with apc.